.orchestra/config/components/revng.yml

```diff
@@ -18,8 +18,8 @@ dependencies:
   - llvm
   - qemu
   - boost
-  - clang-release
   - revng-python-dependencies
+  - toolchain/host/binutils
```

Comment:
This is still WIP, it's part of my branch. I'll do this.

Reply:
Ok